src/__tests__/patients/Patients.test.tsx

```diff
@@ -106,14 +113,20 @@ describe('/patients/edit/:id', () => {
 
     expect(wrapper.find(EditPatient)).toHaveLength(1)
 
-    expect(store.getActions()).toContainEqual(
-      addBreadcrumbs([
+    expect(store.getActions()).toContainEqual({
+      ...addBreadcrumbs([
         { i18nKey: 'patients.label', location: '/patients' },
         { text: 'test test test', location: `/patients/${patient.id}` },
         { i18nKey: 'patients.editPatient', location: `/patients/${patient.id}/edit` },
         { i18nKey: 'dashboard.label', location: '/' },
       ]),
-    )
+      ...changeTitle(
```

Comment:
Not 100% sure this is a great `expect`. Any suggestion how this should be handled?